buildkite/scripts/generate-diff.sh

```diff
@@ -5,13 +5,19 @@ BASE=${BUILDKITE_PULL_REQUEST_BASE_BRANCH:-origin/develop}
 
 # Finds the greatest common commit that is shared between these two branches
 # Or nothing if there isn't one
-COMMIT=$(diff -u <(git rev-list --first-parent HEAD) \
-        <(git rev-list --first-parent $BASE) | \
-        sed -ne 's/^ //p' | head -1)
+# COMMIT=$(diff -u <(git rev-list --first-parent HEAD) \
+#        <(git rev-list --first-parent $BASE) | \
+#        sed -ne 's/^ //p' | head -1)
 
-if [[ $COMMIT != "" ]]; then
-  # Get the files that have changed since that shared commit
-  git diff $COMMIT --name-only
+BASECOMMIT=$(git log $BASE -1 --pretty=format:%H) # Finds the commit hash of HEAD of $BASE branch
+
+# Print it for logging/debugging
+echo "Diffing current commit: ${BUILDKITE_COMMIT} against commit: ${BASECOMMIT} from branch: ${BASE} ."
```

Comment:
Probably want a default for `BUILDKITE_COMMIT` in case the script is not executed in Buildkite (similar to `BUILDKITE_PULL_REQUEST_BASE_BRANCH` above)

Reply:
absolutely. fixed.